deploy/examples/operator.yaml

```diff
@@ -457,6 +457,11 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # To indicate the image pull policy to be applied to the csi driver
+  ROOK_CSI_IMAGE_PULL_POLICY: "IfNotPresent"
```

Comment:
There is a great chance that user might accidentally provide incorrect values for this variable. Lets say `IfnotPresent` instead of `IfNotPresent`. 
Should we handle such cases? I think we should.